screwdriver.yaml

```diff
@@ -5,29 +5,39 @@ shared:
     image: golang
     environment:
         GOPATH: /sd/workspace
+        NAME: logservice
+        OS: linux darwin
```

Comment:
Should add a comment here saying that this will be a list of space separated OS that will be built and released. Otherwise LGMT 👍